charts/artifacthub-repo.yml

```diff
@@ -0,0 +1,4 @@
+repositoryID: 6d08b24c-bd34-41aa-847a-322e28a271b7
+owners:
+  - name: sozercan
+    email: sozercan@gmail.com
```

Comment:
for my own learning, is this approach intended to be a one off? It seems that we'd need to make N changes to support N repositories. 

I have a lack of context here so feel free to pay me no mind.

Also, as a small (future) nit: would it be more valuable to have a "group" or alias as an owner? that way, we can avoid a "single point of failure" SPF and the need to adjust the owner value in the future.

Reply:
Hi @acpana, This file is not GK specific. It's for the Artifact Hub. Please refer https://artifacthub.io/docs/topics/repositories/#verified-publisher